frontend/test/metabase/scenarios/admin/datamodel/metrics.cy.spec.js

```diff
@@ -85,7 +85,7 @@ describe("scenarios > admin > datamodel > metrics", () => {
       );
     });
 
-    it.skip("should have 'Custom expression' in a filter list (metabase#13069)", () => {
+    it("should have 'Custom expression' in a filter list (metabase#13069)", () => {
```

Comment:
Let's completely remove this test now. Repro for 13070 reproduces 13069 line by line.

Reply:
Good point, test is removed.